src/main/java/frc/robot/io/implementations/motor/MotorIOStub.java

```diff
@@ -11,17 +12,25 @@
 public class MotorIOStub extends MotorIOBase {
   private final DCMotorSim motorSim;
   private double appliedVolts = 0;
+  private MotorSimulationSettings simSettings;
 
   public static class MotorSimulationSettings {
     // DC Motor Simulation Settings
     public DCMotor motor = DCMotor.getNEO(1);
     public double moiKgMetersSquared = 1.0;
+
+    // TODO: should not hard-code these values.  Need to be set/passed in from
+    // RobotConfigStub.java somehow.
+    public double forwardLimitPositionDegrees = 90.0;
+    public double reverseLimitPositionDegrees = 10.0;
```

Comment:
The 'native' units for a Motor is currently `radians`.  Also, forward/reverse is ambiguous so I think we should use min/max instead.  I think these should be:
```suggestion

    // TODO: should not hard-code these values.  Need to be set/passed in from
    // RobotConfigStub.java somehow.
    public double maxLimitPositionRads = Units.degreesToRadians(90.0);
    public double minLimitPositionRads = Units.degreesToRadians(10.0);
```

Reply:
Will change to radians.